docs/gateway/azure.md

```diff
@@ -1,54 +1,84 @@
 # Minio Azure Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
-Minio Gateway adds Amazon S3 compatibility to Microsoft Azure Blob Storage.
 
-## Run Minio Gateway for Microsoft Azure Blob Storage
-### Using Docker
-```
+This topic describes how to use Minio Gateway to add Amazon S3 compatibility to Microsoft Azure Blob Storage.
+
+- [Run Minio Gateway for Microsoft Azure Blob Storage](#run-minio-gateway-for-azure) 
+- [Test Using Minio Browser](#test-using-minio-browser) 
+- [Test Using Minio Client](#test-using-minio-client) 
+- [Known Limitations](#known-limitations) 
+- [Explore Further](#explore-further)
+
+## <a name="run-minio-gateway-for-azure"></a> 1. Run Minio Gateway for Microsoft Azure Blob Storage
+
+### 1.1 Run Minio Gateway Using Docker
+
+```sh
 docker run -p 9000:9000 --name azure-s3 \
  -e "MINIO_ACCESS_KEY=azurestorageaccountname" \
  -e "MINIO_SECRET_KEY=azurestorageaccountkey" \
  minio/minio gateway azure
 ```
 
-### Using Binary
-```
+### 1.2 Run Minio Gateway Using the Minio Binary
+
+```sh
 export MINIO_ACCESS_KEY=azureaccountname
 export MINIO_SECRET_KEY=azureaccountkey
 minio gateway azure
 ```
-## Test using Minio Browser
-Minio Gateway comes with an embedded web based object browser. Point your web browser to http://127.0.0.1:9000 to ensure that your server has started successfully.
+
+## <a name="test-using-minio-browser"></a> 2. Test Using Minio Browser
+
+Minio Gateway comes with an embedded web-based object browser that outputs content to http://127.0.0.1:9000. To test that Minio Gateway is running, open a web browser, navigate to http://127.0.0.1:9000, and ensure that the object browser is displayed.
 
 ![Screenshot](https://github.com/minio/minio/blob/master/docs/screenshots/minio-browser-gateway.png?raw=true)
-## Test using Minio Client `mc`
-`mc` provides a modern alternative to UNIX commands such as ls, cat, cp, mirror, diff etc. It supports filesystems and Amazon S3 compatible cloud storage services.
 
-### Configure `mc`
-```
+## <a name="test-using-minio-client"></a> 3. Test Using Minio Client
+
+Minio Client is a command-line tool called `mc` that provides UNIX-like commands for interacting with the server  (e.g. `ls`, `cat`, `cp`, `mirror`, `diff`, `find`, etc.). 
+`mc` supports file systems and Amazon S3-compatible cloud storage services (AWS Signature v2 and v4).
+
+Test `mc` with the gateways using the instructions in the [Minio Client Quickstart Guide](https://docs.minio.io/docs/minio-client-quickstart-guide).
+
+### 3.1 Configure the Gateway Using Minio Client
+
+Use the following command to configure the gateway:
+
+```sh
 mc config host add myazure http://gateway-ip:9000 azureaccountname azureaccountkey
```

Comment:
here is this azureaccountname or azurestorageaccountname ?
and the same for azureaccountkey